### PR TITLE
Make crossdac build more resilient

### PR DESCRIPTION
**Pull request by hoyosjs**

### CODE REVIEW

**Review by mikem8361 — APPROVED** [review]

File: eng/finalize-publish.yml [hunk 1]
```diff
@@ -12,12 +12,27 @@ parameters:
...
+  - job: waitForValidation
+    displayName: Wait for external validation
+    pool: server
+    timeoutInMinutes: 1440 # job times out in 3 days
+    steps:
+    - task: ManualValidation@0
+      timeoutInMinutes: 1440 # task times out in 1 day
+      inputs:
+        notifyUsers: |
+          juhoyosa@microsoft.com
```
Comment (mikem8361) [inline eng/finalize-publish.yml]: Maybe other diagnostics team members should be notified?

Comment (tommcdon) [reply eng/finalize-publish.yml]: > Maybe other diagnostics team members should be notified?

+1.  Please add me too :)

Comment (hoyosjs) [reply eng/finalize-publish.yml]: I considered it, and it's fine as long as whoever does it does the verification that the one that's getting published actually matches the one being published.

Comment (hoyosjs) [inline eng/finalize-publish.yml]:
```suggestion
          clrdiagd@microsoft.com
```

**Pull request merged**